source/initial-dns-seedlist-discovery/tests/uri-with-auth.yml

```diff
@@ -0,0 +1,12 @@
+uri: "mongodb+srv://auser:apass@test1.test.build.10gen.cc/?replicaSet=repl0"
+seeds:
+    - localhost.test.build.10gen.cc:27017
```

Comment:
Should this also have `"localhost.test.build.10gen.cc:27018"`?

Reply:
Ah, yes, thanks for catching this. Looks like I confused the one/two-results-default-port original 👍